src/autopas/utils/StringUtils.h

```diff
@@ -251,6 +251,55 @@ inline std::set<int> parseInts(const std::string &intString) {
   return ints;
 }
 
+template <class N>
+inline std::unique_ptr<autopas::NumberSet<N>> parseNumberSet(const std::string &setString) {
+  std::string str = regexDoubleStr;
+  if (typeid(N) == typeid(int)) {
+    str = regexIntStr;
+  }
```

Comment:
It would be cleaner to handle this inside the `if`-statement below.

Reply:
Yeah had the same thought. But than i need one more if clause afterwards because either way I want to parse the String. But the method has to be chosen. I could changes these though if you like it more